### PR TITLE
Exptype edits

### PR DESCRIPTION
**Pull request by sbreiff**
- added options to suggested enums in ExperimentType schema
- removed experiment_type field from Protocol schema
- added other_protocols field to ExperimentType schema
- moved revlink from ExperimentType to Protocol
- wrote tests for experiment_type calc prop on Protocol item
- added pre-release to list of statuses that allow Download button

### CODE REVIEW

File: src/encoded/types/experiment_type.py [hunk 1]
```diff
@@ -71,18 +68,18 @@ class ExperimentType(Item):
...
-        "description": "Other protocols associated with this experiment type besides the SOP",
-        "type": "string",
-        "linkTo": "Protocol"
-    })
-    def other_protocols(self, request, sop=None):
-        protocol_paths = self.rev_link_atids(request, 'additional_protocols')
-        protocols = [path for path in protocol_paths if path != sop]
-        if protocols:
-            return protocols
+    #
```
Comment (j1z0) [inline src/encoded/types/experiment_type.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4952628)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity decreasing per file
==============================
+ src/encoded/types/experiment_type.py  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4952628&bid=16368429)

**Comment by aschroed** [comment]
Oh and BTW the code changes look good to me.  Did you test trying to grab some info using get_es_metadata to see if circular references were an issue - might not be useful to test on local and I think that a change I made to the script to release experiment sets to avoid ExperimentType being an issue (adding 'experiment_type' to the ignore_field parameter of get_es_metadata is probably not in dcicwrangling master yet.

**Comment by sbreiff** [comment]
> Oh and BTW the code changes look good to me. Did you test trying to grab some info using get_es_metadata to see if circular references were an issue - might not be useful to test on local and I think that a change I made to the script to release experiment sets to avoid ExperimentType being an issue (adding 'experiment_type' to the ignore_field parameter of get_es_metadata is probably not in dcicwrangling master yet.

No, I didn't - but its probably good practice to avoid circular references anyway and I assume that if ExpType has all the direct references and Protocol only has revlinks then this won't be a problem anymore but correct me if I'm wrong.

Also happy to move the types files around as you suggest.

**Comment by aschroed** [comment]
I suspect that experiment_type may still have to be left in the ignore_fields list as it was probably the linked experiment sets that causes it to go haywire - retrieved more than it should and that your changes while improving the situation for protocol/experiment_type references probably won't solve the get_es_metadata problem in that sense but am also not 100% sure.

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/encoded/tests/test_types_protocol.py  3
- src/encoded/types/protocol.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4952628&bid=16368429)

File: src/encoded/types/protocol.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    rev = {
+        'exp_type': ('ExperimentType', 'other_protocols'),
+        'sop_exp': ('ExperimentType', 'sop')
+    }
+
+    @calculated_property(schema={
+        "title": "Display Title",
+        "description": "A calculated title",
+        "type": "string"
+    })
```
Comment (j1z0) [inline src/encoded/types/protocol.py]: Codacy Issue found: [Parameters differ from overridden 'display_title' method](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4952628)

**Pull request merged**